backend/src/main/java/com/library/backend/controller/UserController.java

```diff
@@ -2,23 +2,30 @@
 import com.library.backend.model.User;
 import com.library.backend.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import java.util.Optional;
 
 @RestController
-
+@RequestMapping("/api/users")
 
 public class UserController {
     @Autowired
     private UserService userService;
```

Comment:
Instead of:
@Autowired
private UserService userService;

create the constructor yourself:
private UserService userService

public UserController (UserService userService) {
this.userService = userService;
}

to avoid problems with injections as per SonarCloud:
Field dependency injection should be avoided [java:S6813](https://sonarcloud.io/organizations/sb0208/rules?open=java%3AS6813&rule_key=java%3AS6813)